case-lib/config.sh

```diff
@@ -81,3 +81,6 @@ SOF_TEST_INTERVAL=${SOF_TEST_INTERVAL:-5}
 # Refer to: https://github.com/thesofproject/sof-test/issues/471 and
 # https://github.com/thesofproject/sof-test/issues/913 for more information.
 FALLBACK_TO_PROC=${FALLBACK_TO_PROC:-false}
+
+# Skip the storage checks if they aren't required
+SKIP_STORAGE_CHECKS=${SKIP_STORAGE_CHECKS:-0}
```

Comment:
Change of mind sorry, can you please remove the negation and make this `ENABLE_STORAGE_CHECKS, defaulting to 1`? Later we can just change the default to zero.